File: app/src/am_sangntn/com/example/asian/UserDao.java
package com.example.asian;

import androidx.room.Dao;
import androidx.room.Delete;
import androidx.room.Insert;
import androidx.room.Query;

import java.util.List;

@Dao
public interface UserDao {

    @Insert
    void insertUser(User user);

    @Query("SELECT * FROM user_table")
    List<User> getListUser();

    @Query("DELETE FROM user_table")
    void deleteAll();

    @Delete
    void deleteUser(User user);

}

File: Kotlin/Test/app/src/main/java/com/example/kotlin/model/UserDao.kt
package com.example.kotlin.model

import androidx.lifecycle.LiveData
import androidx.room.Dao
import androidx.room.Delete
import androidx.room.Insert
import androidx.room.OnConflictStrategy
import androidx.room.Query

@Dao
interface UserDao {
    /*@Insert()
    suspend fun insertUser(user: User): Int*/
    @Query("SELECT * FROM users")
    fun getAllUsers(): LiveData<List<User>>
}